src/main/java/seedu/address/logic/parser/ClearCommandParser.java

```diff
@@ -0,0 +1,28 @@
+package seedu.address.logic.parser;
```

Comment:
could add test cases for `ClearCommandParser`

Reply:
Done